app/inject/ProductionModule.java

```diff
@@ -23,7 +25,9 @@
         return seq(
```

Comment:
Unfortunately, it's what Play forces to use. You can check the documentation about it: https://www.playframework.com/documentation/2.4.x/JavaDependencyInjection (section Play Libraries).